### PR TITLE
fix: wrap all touchables in a view

### PR DESCRIPTION
**Pull request by nlewis84**
This fix ensures that Android users will be able to interact with all touchables and this does not change the experience at all for iOS users.

### CODE REVIEW

File: packages/apollos-ui-kit/src/Touchable/index.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const PlatformTouchableWithDefaultProps =
 const Touchable = ({ borderRadius, children, ...otherProps }) => (
   <AndroidWrapper borderRadius={borderRadius}>
     <PlatformTouchableWithDefaultProps {...otherProps}>
-      {children}
+      <View>{children}</View>
```
Comment (nlewis84) [inline packages/apollos-ui-kit/src/Touchable/index.js:52]: @redreceipt This is the only code change.

**Review by redreceipt — APPROVED** [review]

**Comment by nlewis84** [comment]
Hmmm I already updated those `mapview` snaps

**Comment by vinnyjth** [comment]
@nlewis84 conflicts in snaps :) you probably need to update snaps again

**Comment by nlewis84** [comment]
I'm doing ActionTable cross your fingers @vinnyjth 🤞

**Comment by vinnyjth** [comment]
You did it!!!

**Comment by nlewis84** [comment]
> You did it!!!

Whew 😅

**Pull request merged**